tests/integration/conftest.py

```diff
@@ -54,8 +54,34 @@ def requests_timeout():
     yield 15
 
 
-@pytest_asyncio.fixture(scope="module")
-async def app(ops_test: OpsTest, app_name: str, app_config: Dict[str, str], pytestconfig: Config):
+@fixture(scope="module")
+def run_action(ops_test: OpsTest) -> Callable[..., Awaitable[Any]]:
```

Comment:
What is the reason for making this a fixture please? Would this be better coded as a helper function? It seems quite helpful

Reply:
The reason that this is not a helper function is that this function requires access to the `ops_test`, which is a fixture.